features/step_definitions/user_steps.rb

```diff
@@ -13,14 +13,29 @@ def see_journey_picker?
   @see_journey_picker
 end
 
+def page_mapping(page_name)
+  case page_name
+  when 'verify start' then
+    'start'
+  when 'IdP picker' then
```

Comment:
Not to be picky but the correct capitalisation is `IDP`

Reply:
The change has been made as requested.